src/main/java/net/pms/io/WinUtils.java

```diff
@@ -83,7 +83,7 @@ boolean GetVolumeInformationW(
 	public void disableGoToSleep() {
 		// Disable go to sleep (every 40s)
 		if (configuration.isPreventsSleep() && System.currentTimeMillis() - lastDontSleepCall > 40000) {
-			LOGGER.trace("Calling SetThreadExecutionState ES_SYSTEM_REQUIRED");
+			LOGGER.trace("Disable the PC sleeping mode");
```

Comment:
I'd call it something like "Enable Windows sleep mode prevention" instead and corresponding "Disable Windows sleep mode prevention" below. This is more correct, as UMS cannot enable and disable sleep mode which is done by the user in the OS configuration. What UMS can do is to prevent sleep mode while some thread is running.